tests/unit/ui/test_console.py

```diff
@@ -15,6 +15,33 @@ def test_write(capsys: CaptureFixture[str]):
     assert captured.err == f"{message}\n"
 
 
+def test_write_json(capsys: CaptureFixture[str], mocker):
+    """Test that ui.write_json works."""
+    import textwrap
+
+    console = Console(enable=True)
+    mocker.patch.object(console, "isatty", return_value=True)
+    message = {"hello": "world"}
+    console.write_json(message)
+    captured = capsys.readouterr()
+    assert captured.out == textwrap.dedent(
+        """\
+        {
+          "hello": "world"
+        }
+    """
+    )
+
+    mocker.patch.object(console, "isatty", return_value=False)
+    console.write_json(message)
+    captured = capsys.readouterr()
+    assert captured.out == textwrap.dedent(
+        """\
+        {"hello": "world"}
+        """
+    )
```

Comment:
Could you please use `pytest.mark.parametrize` instead?

Reply:
Fixed